components/references.bib

```diff
@@ -539,6 +539,19 @@ @article{McInnes2018-how-umap-works
   url    = {https://umap-learn.readthedocs.io/en/latest/how_umap_works.html#}
 }
 
+@article{Meng2019,
```

Comment:
Surprised to see this added here? I _believe_ this is the paper that's about meta-analysis with QuSAGE..

Reply:
Ah yes, when I looked up the package citation this was one of the sources that were presented (it was third on the list however). I removed this reference and updated it to use the actual R package reference 👍